Implement CRUD "Song" using MVC pattern

SJAmanda opened the pull request:
This Pull Request introduces a CRUD module named Song, implemented with the MVC architecture.
The CRUD functionality includes:

Registering the song name

Storing the album

Recording the artist

Adding the release year

The implementation follows a clean and structured approach, ensuring readability and maintainability.

SJAmanda committed:
A CRUD named Song was created using the MVC model. This CRUD records the song name, album, artist, and year.

SJAmanda committed:
Update db.sql

Se arreglo un error de espacio en la base de datos

SJAmanda committed:
Update db.sql

SJAmanda committed:
Update songDTO.php

Se arreglo un fallo de Enter al final

SJAmanda committed:
Update songDAO.php

Se arreglo un fallo de Enter al final del código

And-Y21 commented on config/config.php:
Este archivo no se tenia que cambiar.

SJAmanda committed:
Update config.php

Edite es archivo para dejarlo en su forma original, elimine el try catch

SJAmanda committed:
Update db.sql

Cambie el nombre de tabla de music a album

SJAmanda committed:
Update songDAO.php

Edite este apartado para cambiar las llamad a la tabla ya que ya no es music ahora es album

SJAmanda committed:
Merge branch 'master' into feature/add-song-crud

And-Y21 commented on database/db.sql:
```diff
@@ -167,5 +184,5 @@ CREATE TABLE song(
     duration_song varchar(50),
     record_song varchar(50),
-	year_launch_song int
+	  year_launch_song int
```
Quitar tab

SJAmanda committed:
Added Album CRUD with functionalities to add song name, artist, album, and year

SJAmanda committed:
Update song.php

I deleted all the CRUD I added

SJAmanda committed:
Update song.php

I deleted all the CRUD I added

SJAmanda committed:
Update songDAO.php

I deleted all the CRUD I added

SJAmanda committed:
Update songDAO.php

I deleted all the CRUD I added

SJAmanda committed:
Update songDTO.php

Remove what I added from my CRUD

SJAmanda committed:
Update songDTO.php

Remove what I added from my CRUD

SJAmanda committed:
Update song.php

Remove what I added from my CRUD

SJAmanda committed:
Update config.php

I deleted this document since it was not supposed to be touched.

SJAmanda committed:
Delete config/config.php

I deleted this document since it was not supposed to be touched.

SJAmanda committed:
Update song.php

Remove the part of my CRUD

SJAmanda committed:
Remove the repeated CRUD song

SJAmanda committed:
Update songDAO.php

Remove what I added about the CRUD

SJAmanda committed:
Update songDTO.php

Remove what I added about the CRUD

SJAmanda committed:
Update song.php

SJAmanda committed:
Remove the repeated CRUD song and add album CRUD

SJAmanda committed:
Remove the repeated CRUD song, menu, db and controller

SJAmanda committed:
Remove the repeated CRUD song, menu and db

SJAmanda committed:
Add an esc at the end of albumDTO

Hyromy commented:
Qpro
